applications/MeshingApplication/custom_utilities/simplex_to_quadratic_tetrahedra_mesh_converter_utility.h

```diff
@@ -156,77 +156,77 @@ class Tetrahedra10MeshConverter : public LocalRefineTetrahedraMesh
     * @param Coord: The compressed matrix containing at (i,j) the id of the node created between nodes i,j
     * @param New_Elements: The new elements created
     * @param interpolate_internal_variables: A boolean that defines if it is necessary to interpolate the internal variables
-    * @return ThisModelPart: The model part of the model (it is the input too)
+    * @return rThisModelPart: The model part of the model (it is the input too)
     */
 
     void EraseOldElementAndCreateNewElement(
-            ModelPart& ThisModelPart,
+            ModelPart& rThisModelPart,
             const compressed_matrix<int>& Coord,
             PointerVector< Element >& NewElements,
             bool interpolate_internal_variables
     ) override
     {
-	ElementsArrayType& rElements = ThisModelPart.Elements();
-	ElementsArrayType::iterator it_begin = rElements.ptr_begin();
-	ElementsArrayType::iterator it_end = rElements.ptr_end();
-
-	const ProcessInfo& rCurrentProcessInfo = ThisModelPart.GetProcessInfo();
-	int edge_ids[6];
-	std::vector<int> node_ids;
-
-    const Element& rElem = KratosComponents<Element>::Get("Element3D10N");
-	for (ElementsArrayType::iterator it = it_begin; it != it_end; ++it)
-	{
-        GlobalPointersVector< Element >& rChildElements = it->GetValue(NEIGHBOUR_ELEMENTS);
-        rChildElements.resize(0);
-
-        CalculateEdges(it->GetGeometry(), Coord, edge_ids, node_ids);
-
-        // Generate the new Tetrahedra3D10 element
-        Tetrahedra3D10<Node<3>> geom = GenerateTetrahedra(ThisModelPart, node_ids);
-        Element::Pointer p_element;
-        p_element = rElem.Create(it->Id(), geom, it->pGetProperties());
-        p_element->Initialize(rCurrentProcessInfo);
-        p_element->InitializeSolutionStep(rCurrentProcessInfo);
-        p_element->FinalizeSolutionStep(rCurrentProcessInfo);
-
-        // Setting the internal variables in the "child" elem (the element replacing the old one)
-        if (interpolate_internal_variables == true)
+        auto& rElements = rThisModelPart.Elements();
```

Comment:
local variables are snake_case (i.e. r_elements) I would suggest you to take a look at [Kratos Style Guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide)